packages/runtime/src/errors.ts

```diff
@@ -62,7 +62,6 @@ export function assertAdaptorError(e: any) {
 // Abstract error supertype
 export class RTError extends Error {
   source = 'runtime';
-  includeStackTrace = false;
```

Comment:
This is incidental and unrelated. I've been seeing it in error reports and it bugs me.